### PR TITLE
Feature/talk page

### PR DESCRIPTION
**Pull request by Watyekele**
### Added html line
I just added a paragraph and heading in the html file.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <h1>Parent Talks</h1>
+  <p>Advise fellow parents</p>
```
Comment (brandie8) [inline index.html:9]: Seen this

Comment (Watyekele) [reply index.html:9]: Oh thank you

**Review by brandie8 — APPROVED** [review]
Saw the change

**Comment by brandie8** [comment]
Saw the commit

**Pull request merged**